.travis.yml

```diff
@@ -13,17 +13,75 @@ python:
   - 3.8
 
 deploy:
-  provider: releases
-  file_glob: true
-  file: dist/*.*
-  skip_cleanup: true
-  draft: true
-  token:
-    secure: V1Ad29+3lIQB1Sxqt8RiNKwferucTIr2mJo1Ka0m8Y1GdscP0ZkvHN4Vzjb0hts+lap8cU24tJ1zq9k22bdsNmoS3/YEg7fwbQSOPCJSYbjwMQeitDc1NP3u1mp5LWiEnQ7cflalIkmWUVpwGbwPSnUQdW3JyheUm4DP6jndGHmv3sC2r4WtZDkVY44SI5OGjvOTStTLnXhVlwXMIBKzw+o9BNdmIcpRvI3UTR+C9mA2yoNE7qgpzALZ4qn7qAmTMJzcVQb22yVZqXVCF8VD72Z66GHx75aatMmsJ/2isOEap19sVYt03fY8lchsTUsdRRCAIInxUvbdfgaEoBWj8Vfv2r/vm5DgWh93amv1RkJdmAYAMTGZG/k20vd7N+sNGKCdAF0W8hj5kriGkNnSQg7dFd+NWrF3pX9XcUkT9IGzv5eT5eYeS+yzgB3YPy8qGKwF5dRkRf7Ylw68j8YOly40rAAfgEx/pcFlYrJh62LDPr5RNhKA679+9UAp0rgkiszNH327bj1i0drqeDIhJAZSXw86B8tJZI1JRBVllKNbwDc5GRFNyms4+pvKCO+lAy5Pcn0bQMUq49qxDsdnM1t2RISivWy5CdkOTNW5WYglhovOSoA8/NjcxmsOpOazCaENEvsB3AogI+LRmpZ7lUZKxjaILuVUM2rwXlKxFB0=
-  on:
-    tags: true
-    all_branches: true
-    condition: $MAKE_STEP == ?dist*
+  - provider: releases
```

Comment:
Just to clarify the role of each block. Would be nice to have a comments about what each deploy block is fow in human word.
e.g.: Push all source, distribution, wheel pacakges for Linux and Windows on github.
Push wheel packages to test.pypi.org when commit is tagged with `*dev`
Push wheel pacakges to pypi.org when commit is tagged.

Reply:
Done in the last commit.